bluezero/adapter.py

```diff
@@ -258,10 +259,13 @@ def _properties_changed(self, interface, changed, invalidated, path):
         Handle DBus PropertiesChanged signal and
         call appropriate user callback
         """
+        macaddr = path.split("/")[-1].replace("dev_", '').replace("_", ":")
```

Comment:
I like how efficient you get the mac address of the device. This could be useful elsewhere so I would if it would be better to make it a function in `dbus_tools` file?

Reply:
yep i can move in tools

Reply:
done